005_team_3_agent_design/005_team_3_agent_design.tex

```diff
@@ -1 +1,194 @@
+
+\subsection{The Agent’s Social Network}
+The agent forms relationships with others surrounding it as it communicate with each other. The agent remembers everyone it has met and assigns a level of friendship to it depending on its interactions. By doing this, the agent forms a social network that will affect its decisions, something that the Theory of Planned Behaviour explores in the “subjective norm” aspect of the theory. In the TPB,  all three main variables are interdependent and for this reason, the agent will have a better or worse opinion of someone it has just met depending on their morality at that point. \par
+The large importance which our agent places on friendship is not only a logical interpretation of the TPB. In fact, it is a notion which philosophers and legislators have repeatedly confirmed throughout the ages. For example, the Aristotelian concept of friendship defends it as “the finest way we exercise practical reasoning”. Quoting Aristotle: “Friendship seems to hold cities together, and lawgivers seem to care for it more than for justice … and when men are friends, they have no need for justice”, friendship is considered the best form of human moral action by The Nicomachean Ethics \cite{sokolowski}.Thus, it is no surprise that friendship is fundamental in the decision-making process of the agent, and also in its responses to communication initiated by other agents. \par 
+To provide an alternative perspective on friendship, McCullough et al. investigate the nature of prosocial action and gratitude’s place in transactions \cite{doi:10.1111/j.1467-8721.2008.00590.x}. Our friendship function allows us to assign a level of gratitude to our agent in response to the prosocial action of the other agents. In this way, we could more effectively alert the agent to the prosocial nature of another’s actions. By utilising the friendship function to, in turn, behave more prosocially to those who have done so to us, we embedded a system of gratitude in our agents’ actions. This allows the agents to be more capable of self-organising.\par 
+We further developed our response to actions taken by foreign agents by evaluating the selflessness of the action taken. That is to say, the more severe the sacrifice another agent makes, the more grateful we become. This is based on McCullough’s definition that one experiences gratitude when “they have benefited from someone’s costly, intentional, voluntary effort on their behalf”. \par 
+Emotions allow us to respond to reality in a self-protective and self-enhancing way \cite{ekman_1992}, it was this response that we wanted our agent to embody so that emotions could govern the evolution of their actions, by either leading them on a self-protective path, if circumstances caused selfish agents to prevail, or a self-enhancing manner, as cooperation would beget further cooperation through our system of gratitude.\par
```

Comment:
```suggestion
Emotions allow us to respond to reality in a self-protective and self-enhancing way \cite{ekman_1992}, it was this response that we wanted the agent to embody so that emotions could govern the evolution of their actions, by either leading them on a self-protective path, if circumstances caused selfish agents to prevail, or a self-enhancing manner, as cooperation would beget further cooperation through the agents' system of gratitude.\par
```